### DISCUSSION TITLE
Accessing the strings from KeyComponentWidget.ui

### DISCUSSION THREAD

**Discussion by StoyanDimitrov** [keepassxreboot/keepassxc#10255] [Q&A] [answered]
There are few strings in `KeyComponentWidget.ui` file that I'm unable to view in the application to get better understanding of what "Key Component" is meant to be. If possible can I get a screenshot of this screen or step by step guide how to get to it? I suppose currently I does not have some sort of dependency in order to access this particular screen. Is it related to SSH Agent?

**Comment by droidmonkey** [comment, marked as answer]
Key Component refers to the underlying UI for the database password / key file / hardware key. Nothing to do with SSH Agent.

**Reply by StoyanDimitrov** [reply]
So is it somewhere around here?
<img src="https://github.com/keepassxreboot/keepassxc/assets/10962709/c96044bf-9dbb-4b17-a967-fde188fe7f94" width="150"/>

**Reply by droidmonkey** [reply]
That's it, yes

**Reply by StoyanDimitrov** [reply]
Is it possible that I mislead you? I'm referring to [KeyComponentWidget.ui](/keepassxreboot/keepassxc/blob/442d65a497161d9a6e3dd4bec5ac1fb62de04f46/src/gui/databasekey/KeyComponentWidget.ui#L41).

**Reply by droidmonkey** [reply]
Well, I certainly have no idea what you are trying to achieve, that would help

**Reply by StoyanDimitrov** [reply]
I'm translator. There are few strings in this file that I'm unable to view in the application to get better understanding of what "Key Component" is meant to be. If possible can I get a screenshot of this screen/widget or step by step guide how to get to that part of the application?

**Reply by droidmonkey** [reply]
It's probably an accessibility string that is not visible on the ui. Just translate it like I mentioned above.
